src/main/webapp/resources/js/pages/admin/components/announcements/EditAnnouncementModal.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useRef } from "react";
+import { Checkbox, Form, Input, Modal, Space } from "antd";
+import { IconEdit } from "../../../../components/icons/Icons";
+import { FONT_COLOR_PRIMARY } from "../../../../styles/fonts";
+import { MarkdownEditor } from "../../../../components/markdown/MarkdownEditor";
+
```

Comment:
Since this is almost exactly the same as the CreateAnnouncementModal do you think you could refactor to use only one?

Reply:
Yes.